Projects/DbCLI/CreateDb.py

```diff
@@ -74,6 +74,7 @@
 
 parser = argparse.ArgumentParser(usage)
 
+parser.add_argument('filename')
```

Comment:
This adds the positional argument. As it is written, the argument is required. This is not the really the case. Check the documentation of `argparse` on how to make it optional.